src/main/java/org/broadinstitute/consent/http/service/DatasetService.java

```diff
@@ -204,7 +222,7 @@ public Dataset approveDataset(Dataset dataset, User user, Boolean approval) {
   private void sendDatasetApprovalNotificationEmail(Dataset dataset, User user, Boolean approval)
       throws Exception {
     Dac dac = dacDAO.findById(dataset.getDacId());
-    if (approval) {
+    if (Boolean.TRUE.equals(approval)) {
```

Comment:
This form is only needed because `approval` is not of type `boolean`. Since we know it's not null (it's validated in `DacResource.approveDataset()` you can change its type to `boolean` here and in `approveDataset()` above so this isn't required.